### DISCUSSION TITLE
Cannot Set Foreign key to allowNull : true when it was Initialised with allowNull: false

### DISCUSSION THREAD

**Discussion by prkhrv** [sequelize/sequelize#16049] [Help & Questions] [answered]
I have a Table 

initial
```
physician_id:{
        type:Sequelize.UUID, // Physician ID
        allowNull:false,
        references:{
            model:'physicians-new',
            key:'user_id'
        }
    },

PatientVisit.belongsTo(models.physician, {foreignKey: 'physician_id', as: 'physician',onDelete:'cascade'});
```

Now I want to set allowNull: true. But it's not letting me to create a row with physician_id : null . Do I need to drop the table ?

**Comment by ephys** [comment, marked as answer]
You need to [migrate](https://sequelize.org/docs/v6/other-topics/migrations/) your table, edit it manually, or drop your table if you're not yet in production and don't care about your data

**Reply by prkhrv** [reply]
Thank you for your response, how do I create migrations for existing models?

**Reply by ephys** [reply]
See https://sequelize.org/docs/v6/other-topics/migrations/#migration-skeleton
